### ISSUE TITLE
[Bug] Telemetry EACCESS

### ISSUE THREAD

**Issue by bertho-zero** [yarnpkg/berry#1780]
- [ ] I'd be willing to implement a fix

**Describe the bug**

I updated Yarn to version 2.2 and since then I have this error in a docker container that launches a node server:

```
Internal Error: EACCES: permission denied, mkdir '/.yarn/berry'
    at Object.mkdirSync (fs.js:947:3)
    at a.mkdirSync (/root/oa/.yarn/releases/yarn-2.2.1.cjs:2:393655)
     at a.sendReport (/root/oa/.yarn/releases/yarn-2.2.1.cjs:2:344739)
    at new a (/root/oa/.yarn/releases/yarn-2.2.1.cjs:2:343597)
    at /root/oa/.yarn/releases/yarn-2.2.1.cjs:2:239930
    at async r (/root/oa/.yarn/releases/yarn-2.2.1.cjs:2:239387)
```

It looks like it came from sending the report for telemetry.

If I add the creation of this folder in my Dockerfile then I get this error:

```
Internal Error: EACCES: permission denied, open '/.yarn/berry/telemetry.json'
    at Object.openSync (fs.js:465:3)
    at Object.writeFileSync (fs.js:1416:35)
    at a.writeFileSync (/root/oa/.yarn/releases/yarn-2.2.1.cjs:2:392656)
    at a.writeJsonSync (/root/oa/.yarn/releases/yarn-2.2.1.cjs:2:386801)
    at a.sendReport (/root/oa/.yarn/releases/yarn-2.2.1.cjs:2:344787)
    at new a (/root/oa/.yarn/releases/yarn-2.2.1.cjs:2:343597)
    at /root/oa/.yarn/releases/yarn-2.2.1.cjs:2:239930
    at async r (/root/oa/.yarn/releases/yarn-2.2.1.cjs:2:239387)

```

**Environment if relevant (please complete the following information):**

 - OS: Linux
 - Node version 14.2.0
 - Yarn version 2.2.1

**Comment by arcanis** [comment]
Will check that and release a fix - as a quick workaround, you can set the `YARN_ENABLE_TELEMETRY=0` or `CI=1` variables in your image

**Comment by bertho-zero** [comment]
I feel like a piece of path is missing, Yarn is trying to create a `.yarn` folder at `/`.

**Comment by arcanis** [comment]
@bertho-zero it uses the home folder, which I suspect in your Docker instance is (incorrectly?) configured to be `/`

**Comment by bertho-zero** [comment]
@arcanis Sorry for deleting, I hadn't seen your answer and noticed that $HOME is on / in my Dokcer instance.

**Comment by arcanis** [comment]
Released in 2.2.2 👍
